backend/migrations/versions/8b425b454a86_fix_created_at_field.py

```diff
@@ -19,10 +19,16 @@
 
 
 def upgrade():
-    query = f"UPDATE score_delegation SET created_at = make_date(2024, 7, 17) WHERE hashed_addr IN ('{HASH1}', '{HASH2}', '{HASH3}');"
-    op.execute(query)
+    return
```

Comment:
migrations are applied only once in a given system and it's "event sourced" so i think we may leave it here for the granularity in the history of migrations?

Reply:
TL;DR: bad commit, fixed.

Longer verison. This migration uses make_date, which is not implemented by sqlite. I've turned that off in sqlite env, but still had some strange leftover in db state. I've deleted this migration while debugging the issue and accidentally committed.